.claude/experiences/research/score-bar-chart-handicap-highlighting-bug.md

```
@@ -0,0 +1,1102 @@
+  return rounds.sort((a, b) => a.scoreDifferential - b.scoreDifferential);
+}
+
+// AFTER:
+if (rounds.length <= 5) {
+  return rounds
+    .sort((a, b) => a.scoreDifferential - b.scoreDifferential)
+    .slice(0, 1);
+}
+```
+
+**2. Changed Homepage Chart to Display Differentials**
```

markdownlint-cli2 (0.18.1)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)